example-application/api-under-test.js

```diff
@@ -4,7 +4,7 @@ const axios = require("axios");
 const bodyParser = require("body-parser");
 const mailer = require("./libraries/mailer");
 const OrderRepository = require("./data-access/order-repository");
-
+const PubsubHandler = require("../various-receipes/message-queue/src/pubsub");
```

Comment:
I think the pubsub code should be in the recipe and the example application should stay clean and simple.
Am I missing something?

Reply:
Yeah, you are right, doing the change right now!